cartoframes/data/dataset.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, table_name=None, schema=None,
                  query=None, df=None, gdf=None,
                  state=None, is_saved_in_carto=False, context=None):
         from ..auth import _default_context
-        self._cc = context or _default_context
+        self._c = context or _default_context
```

Comment:
I know `cc` wasn't the best name either, but how about `_con` here too? It improves readability if a user isn't familiar with the code base. I think pep8 will not like variable names like this too.